ZLocation/ZLocation.LiteDB.psd1

```diff
@@ -0,0 +1,14 @@
+@{
+ModuleVersion = '1.0'
+GUID = 'c9e37f57-8308-441b-adb9-fb98f6bbc6c9'
+RequiredAssemblies = @("$PSScriptRoot/LiteDB/LiteDB.dll")
+RootModule = 'ZLocation.LiteDB.psm1'
+FunctionsToExport = '*'
+CmdletsToExport = @()
+VariablesToExport = @()
+AliasesToExport = @()
+PrivateData = @{
+    PSData = @{}
+}
+}
+
```

Comment:
Since this is an internal module for Zlocation, I don't think there is a need to create a proper psd1 - just psm1 will do.

Reply:
At least one of the .psd1s is necessary to load LiteDB.dll, but you're right, we could remove this one.